week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -8,27 +8,26 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? (2 6 4 9 6 13 8 )
+  4. How many times is "f1" called? 2
+  5. How many times is "f2" called? 5
+  6. What value does the "a" parameter take in the first "f1" call? 6
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? 6
 */
 
 let x = 2;
 let a = 6;
 
-const f1 = function(a, b) {
+const f1 = function (a, b) {
   return a + b;
 };
 
-const f2 = function(a, b) {
+const f2 = function (a, b) {
   return a + b + x;
 };
 
 console.log(x);
 console.log(a);
-console.log(b);
```

Comment:
Good - you've spotted that you need to remove the `b` variable as Node will throw a `ReferenceError`